pkg/router/httpTriggers.go

```diff
@@ -46,6 +47,7 @@ type HTTPTriggerSet struct {
 	*functionServiceMap
 	*mutableRouter
 
+	wg                         sync.WaitGroup
```

Comment:
where are we planning to use this waitgroup?

Reply:
Removed it planning to add in separate PR